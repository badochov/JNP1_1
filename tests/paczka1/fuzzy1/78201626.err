Error in line 10: ApyDq A928 13,3
Error in line 1: U2ULbA6asf A636 10,7
Error in line 20: U2ULbA6asf A928 72,5
Error in line 19: MTNB5lHxOx A636 331,8
Error in line 23: U2ULbA6asf S472 81,8
Error in line 32: U2ULbA6asf S720 25,5
Error in line 38: U2ULbA6asf S472 15,7
