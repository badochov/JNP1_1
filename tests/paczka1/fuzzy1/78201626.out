A928 A 94,2
ApyDq S 0,3
MTNB5lHxOx A 82,9
r9zo4YZ A 154,6
S472 0,3
A636 242,4
A928 89,3
A928 A 94,2 S 263,2
ApyDq A 108,3 S 0,3
MTNB5lHxOx A 83,2
r9zo4YZ A 159,4
S472 0,3
A636 355,5
S720 263,2
A928 89,6
